### ISSUE TITLE
vicadmin: Chrome browser uses high CPU tailing logs

### ISSUE THREAD

**Issue by corrieb** [vmware/vic#3043]
Problem evident in 0.7.0 tagged build

When debugging an issue earlier today, I noticed that tailing the logs in vicadmin caused the fans to start whirring on my laptop. Turns out Chrome is using a lot of CPU for a small amount of output in vicadmin (update every couple seconds). 

You wouldn't normally notice this because vicadmin doesn't put out much output. However, I've tried various ways of triggering output (Eg. tailing the vicadmin logs in another tab and exercising vicadmin server) and you get the same problem.

Haven't tried in different browsers, but I suspect it'll be the same. Obviously this is going to need some client side debugging which is outside of my skill set.

**Comment by hickeng** [comment]
I do not see this issue with up-to-date chrome on Windows.

We are not doing anything complex with the log tail - it's just an unterminated http response.
You can try opening the chrome Developer Tools (More Tools->Developer Tools) to see if there's anything obvious there.
Also, please check it's the log tab that's consuming CPU using the chrome task manager.

**Comment by corrieb** [comment]
I reproduced on Chrome after shutting it down and restarting it running only a single tab with vicadmin. CPU rose slowly correspondent seemingly with the amount of text that accumulated in the browser window until it consumed 100% of one CPU. Developer tools didn't reveal anything obvious.

Doesn't appear to be a problem on Firefox.

**Comment by mdubya66** [comment]
I just ran a loop of docker pull and docker rmi's and left the docker personality log streaming in chrome. The main Google Chrome process stayed around 3-4% and there was 1 Google Chrome Helper process that hovered between 30 and 60%. When I switched to a new tab and reloaded the H5 client in parallel the main process cpu jumped to 125% for a bit and 5 new helpers jumped up as well. once loaded things dropped back down to 3-4% on the main thread. Closing as CNR.
